mozdownload/scraper.py

```diff
@@ -721,6 +751,28 @@ def __init__(self, branch='mozilla-central', build_number=None, date=None,
 
     def get_build_info(self):
         """Define additional build information."""
+        # Retrieve build by revision
```

Comment:
If we do that the `Scraper` class would also make it available for all other types of scrapers, which is not wanted. We could think about a better inheritance in a follow-up issue. Whereby the big refactoring is still in-front of us. Would that be fine?